_specs/ecip-1109.md

```diff
@@ -32,12 +32,9 @@ Beacon chain withdrawals are omitted at this time as they are not applicable for
 - EIP 3855 "PUSH0 instruction": Introduces the PUSH0 (0x5f) instruction, which pushes the constant value 0 onto the stack.
 - EIP-3860 "Limit and meter initcode": Limits the maximum size of initcode to 49152 bytes and apply extra gas cost of 2 for every 32-byte chunk of initcode
 
-##### Compatible inclusion
-- EIP 4399 "Supplant DIFFICULTY opcode with PREVRANDAO": Historicaly, the DIFFICULTY (0X44) instruction was used as a source of pseudo randomness. With the switch to Proof of Stake, Ethereum removed the work difficulty so, as a replacement, they now expose a [RANDAO](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/block-proposal/#random-selection) value taken from the beacon chain. Since v0.8.18, Solidity [deprecates](https://docs.soliditylang.org/en/v0.8.18/cheatsheet.html#global-variables) the `block.difficulty` and replaces it with `block.prevrandao` but this syntactic change still compiles to the same EVM opcode, i.e. `0x44`, which in the Ethereum Classic network will continue to work with the aforementioned _block difficulty_.
-
 ##### Omitted
-- EIP 4895 "Beacon chain push withdrawals as operations": In Proof of Stake networks, enable validator withdrawals from the beacon chain to the EVM
-- Ethereum Classic network only grants rewards through Proof of Work mining on the canonical chain, there are no validators, so this doesn't apply.
+- EIP 4399 "Supplant DIFFICULTY opcode with PREVRANDAO": Historicaly, the DIFFICULTY (0X44) instruction was used as a source of pseudo randomness. With the switch to Proof of Stake, Ethereum removed the work difficulty so, as a replacement, they now expose a [RANDAO](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/block-proposal/#random-selection) value taken from the beacon chain. Since v0.8.18, Solidity [deprecates](https://docs.soliditylang.org/en/v0.8.18/cheatsheet.html#global-variables) `block.difficulty` and replaces it with `block.prevrandao` but this syntactic change still compiles to the same EVM opcode, i.e. `0x44`, which in the Ethereum Classic network will continue to work with the aforementioned _block difficulty_.
```

Comment:
```suggestion
- EIP 4399 "Supplant DIFFICULTY opcode with PREVRANDAO": Historically, the DIFFICULTY (0X44) instruction was used as a source of pseudo randomness. With the switch to Proof of Stake, Ethereum removed the work difficulty so, as a replacement, they now expose a [RANDAO](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/block-proposal/#random-selection) value taken from the beacon chain. Since v0.8.18, Solidity [deprecates](https://docs.soliditylang.org/en/v0.8.18/cheatsheet.html#global-variables) `block.difficulty` and replaces it with `block.prevrandao` but this syntactic change still compiles to the same EVM opcode, i.e. `0x44`, which in the Ethereum Classic network will continue to work with the aforementioned _block difficulty_.
```